bookkeeper-server/src/main/java/org/apache/bookkeeper/client/AsyncCallback.java

```diff
@@ -48,6 +48,24 @@
          *          context object
          */
         void addComplete(int rc, LedgerHandle lh, long entryId, Object ctx);
+
+        /**
+         * Callback declaration which additionally passes quorum write complete latency.
+         *
+         * @param rc
+         *          return code
+         * @param lh
+         *          ledger handle
+         * @param entryId
+         *          entry identifier
+         * @param qwcLatency
+         *          QuorumWriteComplete Latency
+         * @param ctx
+         *          context object
+         */
+        default void addCompleteAdv(int rc, LedgerHandle lh, long entryId, long qwcLatency, Object ctx) {
```

Comment:
I wonder if it's possible to put the Stability.Evolving annotation on a single method.

Reply:
Will do